### PR TITLE
Feat/custom colors

### PR DESCRIPTION
**Pull request by esoubihe**
Merge #174 into this branch first.

Replaces tailwind colors with our own custom colors. This makes Metal UI a lot more opinionated, especially since we don't plan on supporting Theming.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -7,6 +7,46 @@ module.exports = {
...
+      selection: "#79FFE1",
+      secondary: "#666666",
+      link: "#0070F3",
+      border: "#EAEAEA",
+      code: "#F81CE5",
+      cyan: "#50E3C2",
+      purple: "#F81CE5",
+      alert: "#FF0080",
+      violet: "#7928CA",
+      accents: {
```
Comment (n23daniel) [inline tailwind.config.js:25]: Having only this one at plural feels a bit weird to me as it's not so consistent with all others which are defined as singular. But I can see Geist uses the accent naming at plural so it also makes sense to have it this way - https://react.geist-ui.dev/en-us/guide/colors

**Review by n23daniel — APPROVED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (esoubihe) [reply tailwind.config.js:25]: We can do it eitherway. Your call.

Comment (n23daniel) [reply tailwind.config.js:25]: If you also think singular makes more sense lets go that way

Comment (esoubihe) [reply tailwind.config.js]: @n23daniel all set

Comment (n23daniel) [reply tailwind.config.js]: looks good. I think we can move forward with this one 🙂

**Pull request merged**